response_operations_ui/views/collection_exercise.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from flask import render_template, request
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.controllers import collection_exercise_controllers
+from response_operations_ui.controllers import collection_instrument_controllers
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@app.route('/surveys/<short_name>/<period>', methods=['GET'])
+def view_collection_exercise(short_name, period):
+    ce_details = collection_exercise_controllers.get_collection_exercise(short_name, period)
+    cis = collection_instrument_controllers.get_collection_instruments(short_name, period)
```

Comment:
I think there's a question around whether getting `cis` should be a separate call or if it should be returned as part of `ce_details` from the original backstage call. 
The pattern we've been using up til now would be to return it in the same call